pkg/ctl/utils/describe_stacks.go

```diff
@@ -71,6 +86,14 @@ func doDescribeStacksCmd(cmd *cmdutils.Cmd, all, events, trail bool) error {
 		logger.Warning("only %d stacks found, for a ready-to-use cluster there should be at least 2", len(stacks))
 	}
 
+	if json {
```

Comment:
We should either support the `all`, `events` and `trail` options for JSON (and YAML) output types, or add validation to disallow their usage when `--output` is specified.

Reply:
@cPu1 which one do you prefer?